llvm/test/CodeGen/AMDGPU/extract-lowbits.ll

```diff
@@ -150,39 +150,22 @@ define i32 @bzhi32_c4_commutative(i32 %val, i32 %numlowbits) nounwind {
 ; ---------------------------------------------------------------------------- ;
 
 define i32 @bzhi32_d0(i32 %val, i32 %numlowbits) nounwind {
-; GCN-LABEL: bzhi32_d0:
```

Comment:
Should keep the test, and also add a variant with canonical IR

Reply:
This test is kept now, thanks.

I think the canonical IR test is there in the test:
define i32 @bzhi32_d0(i32 %val, i32 %numlowbits) nounwind {
  %numhighbits = sub i32 32, %numlowbits
  %highbitscleared = shl i32 %val, %numhighbits
  %masked = lshr i32 %highbitscleared, %numhighbits
  ret i32 %masked
}